lib/open_project/xls_export/hooks/work_package_hook.rb

```diff
@@ -2,12 +2,22 @@ module PrintableIssues
   class IssueHook  < Redmine::Hook::ViewListener
     # Add XLS format link below issue list
     def view_work_packages_index_other_formats(context)
```

Comment:
@furinvader @oliverguenther I've tried adding a new export link for "XLS with relations" here but it is obviously not used at all. Is this by any chance hard coded in the core or some other plugin? At I can't find any code in the actual XLS plugin that seems to be adding these entries.